e2e/cypress/test_module/cypress/features/product-page/ProductsPage.feature

```diff
@@ -0,0 +1,8 @@
+Feature: |Products Page|
```

Comment:
Change folder name to `products-page` instead of current `product-page`

Reply:
Done